pkg/authenticate/authenticate.go

```diff
@@ -97,7 +97,7 @@ func CreateClientFromConfigFile(configFile, clusterCtx string) (clientset *kuber
 // Kubernetes clientset.
 func CreateClientFromS3Bucket(bucket, s3FileName, region, clusterCtx string) (clientset *kubernetes.Clientset, err error) {
 	configFileLocation := filepath.Join("/", "tmp", "config")
```

Comment:
@vijay-veeranki , is it ok to use the s3FileName argument when buidling the configFileLocation?

Reply:
so the line would be `configFileLocation := filepath.Join("/", "tmp", s3Filename)`